runtime/str_test.go

```diff
@@ -347,6 +347,18 @@ func TestStrMethods(t *testing.T) {
 		{"find", wrapArgs("bar", "a", 0, -1), NewInt(1).ToObject(), nil},
 		{"find", wrapArgs("foo", newTestTuple("barfoo", "oo").ToObject()), nil, mustCreateException(TypeErrorType, "'find/index' requires a 'str' object but received a 'tuple'")},
 		{"find", wrapArgs("foo", 123), nil, mustCreateException(TypeErrorType, "'find/index' requires a 'str' object but received a 'int'")},
+		{"isalnum", wrapArgs("123abc"), True.ToObject(), nil},
+		{"isalnum", wrapArgs(""), False.ToObject(), nil},
+		{"isalnum", wrapArgs("#$%"), False.ToObject(), nil},
```

Comment:
Also include a test case where there some some characters are in alnum but others aren't, e.g. "abc#123". Same idea for isalpha and isdigit.